flumeDevice

```diff
@@ -323,7 +324,8 @@ def queryDevice()
 
     // make sure to check for missing setting (pre-1.2.0 update)
     flowAlarmLookback = flowAlarmLookback ?: 5
-    
+    flowAlarmTriggerTime = flowAlarmTriggerTime ?: flowAlarmTrigger * refreshInterval
+
```

Comment:
I think, based on your example, this handles the not-defined use case of new config?